application/manage.py

```diff
@@ -0,0 +1,80 @@
+from psycopg2 import connect
+
+dbname = 'ridemyway'
+user = 'ridemyway'
+host = 'localhost'
+password = 'ridemyway'
+
+
+class db():
+
+    def connect(self, dbname):
+        dbname = dbname
+        con = connect(database=dbname, user=user, host=host, password=password)
+        con.autocommit = True
+        return con
+
+    def create_all(self, dbname=dbname):
+        # Create all tables here
+        con = self.connect(dbname)
+        commands = (
+            'DROP TABLE "users" CASCADE',
+            'CREATE TABLE users (user_id serial PRIMARY KEY, \
+                        username varchar(255), \
+                        email varchar(50) NOT NULL, \
+                        password varchar(255) NOT NULL, \
+                        phone varchar(255) NOT NULL, driver boolean )',
+
+            'DROP TABLE "rides" CASCADE',
+            'CREATE TABLE rides (ride_id serial PRIMARY KEY, \
+                       owner_id serial, \
+                       destination varchar(255), \
+                       start_time varchar(50) NOT NULL, \
+                       route varchar(255) NOT NULL, \
+                       available_space Int NOT NULL, \
+                       FOREIGN KEY (owner_id) REFERENCES users(user_id) )'
+        )
+
+        try:
+            cur = con.cursor()
```

Comment:
Please consider using self-describing variable names, remember other people might work with your codebase in future. How will they identify these variables?

Reply:
Ok, working on that.